app/components/alert_icon_component.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if tag_options[:size].present? %>
+    <%= image_tag(source, alt: alt_text, **tag_options, class: css_class) %>
```

Comment:
I think the problem is that if we `AlertIconComponent.new(width: 20, height: 20)` this would end up rendering the image with the default width and height values of 88 (since **size_attributes is expanded after) instead of the ones passed in, right?

Reply:
Yeah, that's true. There'd be some ways we could address that though, maybe like...

```
{
  width: DEFAULT_WIDTH,
  height: DEFAULT_HEIGHT,
  **tag_options.slice(:width, :height),
}
```

Reply:
See what you think of 5ee77b204 (I already had this going)

Reply:
> See what you think of [5ee77b2](https://github.com/18F/identity-idp/commit/5ee77b2045435af30e49d188c8b9e51d670a1403) (I already had this going)

Just the same minor concern as noted in https://github.com/18F/identity-idp/pull/6983#discussion_r980017071 of using `tag_options` as anything other than render-provided additional attributes (by assigning the default into it).

Reply:
Do we need `size` support anymore since https://github.com/18F/identity-idp/pull/6983#discussion_r980343448 was addressed?

Reply:
Agreed